public/js/index.js

```diff
@@ -111,7 +111,9 @@ console.log(localStorage.getItem('theme'))
 
 // Panic
 document.addEventListener('keydown', async (e) => {
-  if (localStorage.getItem('panickey') && localStorage.getItem('panickey') == e.key) window.parent.window.location.replace(localStorage.getItem('panicurl') || 'https://classroom.google.com/h')
-})
+  let panicurl = localStorage.getItem('panicurl'); 
+  panicurl = (panicurl && (!panicurl.includes("https://") && !panicurl.includes("http://"))) ? "https://"+panicurl : panicurl; 
```

Comment:
This should be formatted better (spaces between plus, etc). I note that you could save an 'includes' statement by using a regexp and the `test` function. Overall could be done better, 6/10.

Reply:
Okay well to be fair I wasn't competing for a coding competition and trying to minify the code to the exact required lines and pretty spaces, I was simply just pitching an idea and I also decided to stack the code on one line since it was formatted that way originally. I wasn't formatting it for read quality, and the original code was also not that way. Not like every line needs to be fully minified and perfect, i'm sure you could fine multiple errors looking in files. lol

Reply:
> This should be formatted better (spaces between plus, etc). I note that you could save an 'includes' statement by using a regexp and the `test` function. Overall could be done better, 6/10.

I also simply left the first line of the panic code the part for the listener since I was not trying to rewrite it all, just add a nice function lol

Reply:
hey but at the end of the day i'm all for learning new methods and ways others do things, sorry if the first reply's came off rude.

Reply:
If you can i'd be more than happy to see how you would've went about it.